crates/matrix-sdk-base/src/sync.rs

```diff
@@ -215,7 +215,7 @@ impl LeftRoomUpdate {
 #[cfg(not(tarpaulin_include))]
 impl fmt::Debug for LeftRoomUpdate {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("JoinedRoom")
+        f.debug_struct("LeftRoom")
```

Comment:
```suggestion
        f.debug_struct("LeftRoomUpdate")
```

Reply:
lulz

Reply:
Err, I kept it to be consistent with the `JoinedRoomUpdate` variant.

Reply:
I'd rather have us use the real full name everywhere, but not a strong opinion here.

Reply:
I don't really care, I just assumed that the `JoinedRoomUpdate` was using the short-hand for a reason.